examples/src/main/java/io/dapr/examples/exception/Client.java

```diff
@@ -35,11 +40,39 @@ public static void main(String[] args) throws Exception {
     try (DaprClient client = new DaprClientBuilder().build()) {
 
       try {
-        client.getState("Unknown state store", "myKey", String.class).block();
+        client.publishEvent("", "", "").block();
+        // client.publishEvent("fake", "fake", "someData").block();
+        // client.getState("Unknown state store", "myKey", String.class).block();
       } catch (DaprException exception) {
         System.out.println("Error code: " + exception.getErrorCode());
         System.out.println("Error message: " + exception.getMessage());
 
+        try {
```

Comment:
This is too complex for users. Can we have an opinionated structure that encapsulates this into the DaprException object?

Reply:
See the latest version with the "opinionated" layer I added.